src/pcp/summary/pcp-summary.sh

```
@@ -462,5 +472,52 @@ then
     fi
 fi
 
+# sysstat/sadc: from live pmimport.*
+_pmimport_tool()
+{
+    # $1 = tool name (instance), $2 = field name
+    [ -f "$tmp/pmimport_$2" ] || return
+    $PCP_AWK_PROG -v tool="$1" '$2==tool{gsub(/"/,"",$3); print $3}' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)